components/counter/src/index.js

```diff
@@ -0,0 +1,3 @@
+import { AuroCounter } from './auro-counter.js';
+
+AuroCounter.register();
```

Comment:
it's calling undefined function I think? 
Also, AuruCouterGroup's registration would be needed.

Reply:
updated